### PR TITLE
updating order of functions, minor tweaks to word usage

### PR DESCRIPTION
**Pull request by benspector3**
- Changed the order of functions
    - Moved indexOf and contains to the top
    - Moved unique to directly follow each
- Removed () at the end of each function description to avoid confusion when defining functions as properties of _ object
    - Example: _.indexOf() = function() { }
- Changed wording for arguments of indexOf and typeOf from “anything” to “value”
- Changed wording from “Gotchas” to “Edge Cases”

- Some things I noticed...
    - Contains test doesn’t test for use of the ternary operator
    - Reject doesn’t test use of _.filter
    - Pluck doesn’t test use of _.map

### CODE REVIEW

File: underpants.js [hunk 1]
```diff
@@ -194,7 +210,7 @@ var _ = {};
 */
 
 
-/** _.pluck()
+/** _.pluck
```
Comment (andyn190) [inline underpants.js]: I think pluck should move to the top before filter if we want to keep all of the functions/higher order functions together.

File: underpants.js [hunk 2]
```diff
@@ -98,32 +131,26 @@ var _ = {};
 */
 
 
-/** _.indexOf()
+/** _.unique
```
[the remark this answers is not in the document]
Comment (benspector3) [reply underpants.js]: The reason for unique being here is similar, it can be solved using _.each so it should follow each

File: underpants.js (hunk 1 above)
Comment (benspector3) [reply underpants.js]: The reason for pluck being where it is is that we want students to use _.map() to solve it so it should follow map. However, the tests currently don't require them to do so. I can change the language in the description to ask that if they solved it without _.map to do so as a bonus challenge?

Comment (andyn190) [reply underpants.js]: I think we should add the test to make sure they are using _.map, it's pretty important for people to understand functional programming with the use of functions they just built.

File: underpants.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (andyn190) [reply underpants.js]: Got it.
